src/frontend/src/env/networks.eth.env.ts

```diff
@@ -2,7 +2,7 @@ import type { Networkish } from '@ethersproject/networks';
 import { Network } from 'alchemy-sdk';
 
 export const ETH_MAINNET_ENABLED =
-	JSON.parse(import.meta.env.VITE_ETHEREUM_MAINNET ?? false) === true;
+	JSON.parse(import.meta.env.VITE_ETHEREUM_MAINNET_DISABLED ?? true) === false;
```

Comment:
With this change of approach, shouldn't we interpret the abscence of the environment flag as the feature being enabled?

I mean default to `false`:

```
JSON.parse(VITE_ETHEREUM_MAINNET_DISABLED ?? false) === false;
```

If flag DISABLE not set => feature is enabled
If flag DISABLE is set to false => feature is enabled
If flag DISABLE set to true => feature is disabled

WDYT?

Reply:
you are totally right! i confused the logic